core/src/main/scala/ai/h2o/sparkling/backend/utils/ProgressBar.scala

```diff
@@ -0,0 +1,24 @@
+package ai.h2o.sparkling.backend.utils
+
+object ProgressBar {
+  private final val BarLength = 50
+  private final val CarriageReturn = "\r"
+  private final val Empty = "_"
+  private final val Filled = "█"
+  private final val Side = "|"
+
+  private[sparkling] def printProgressBar(progress: Float, leaveTheProgressBarVisible: Boolean = false): Unit = {
+    val crOrNewline = if (leaveTheProgressBarVisible) System.lineSeparator else CarriageReturn
+    System.err.print(CarriageReturn + renderProgressBar(progress) + crOrNewline)
```

Comment:
Wouldn't it be better to print progressbar to stdout insteadoff stderr?

Reply:
There are two reasons for why I did that:
1. I noticed that Spark does that with it's progress bar and I think it works well so I did the same thing
2. `java.lang.System.err` javadoc says:
> By convention, this output stream is used to display error messages or other information that should come to the immediate attention of a user even if the principal output stream, the value of the variable out, has been redirected to a file or other destination that is typically not continuously monitored.

even so I think changing that to stdout should be ok if you think it would be better

Reply:
My motivation to move the progress to stdout is based on the below points:
- H2O python and R client prints progressbar to stdout
- Spark prints out to stderr also other logging messages that were captured via `org.apache.spark.internal.Logging`. So the stderr can contain lots of messages coming from Spark depending the logging level and the information about the progress bar might get easily lost.
- Is stderr visible in Jupyter/Databricks notebook?

Please, correct me if any of the points is not true.

Reply:
now it's clear, thank you, I'm checking right now how it works in the jupyter notebook and will implement accordingly

Reply:
![image](https://user-images.githubusercontent.com/8573352/144334571-50065735-c02e-44c0-9ded-ddf29f372176.png)
so on stderr it looked like that, changing to stdout as requested

Reply:
<img width="1344" alt="image" src="https://user-images.githubusercontent.com/8573352/144335193-d1954c11-a923-4533-85b0-270257a4049b.png">
and that's the current version I'm pushing right now

Reply:
This looks great!!!